kernel/kernel-api/src/main/java/io/delta/kernel/internal/snapshot/SnapshotManager.java

```diff
@@ -255,6 +256,11 @@ public LogSegment getLogSegmentForVersion(Engine engine, Optional<Long> versionT
     final Optional<Long> startCheckpointVersionOpt =
         getStartCheckpointVersion(engine, versionToLoadOpt);
 
+    logger.info(
```

Comment:
it looks like we already have log this in `getStartCheckpointVersion`. Is this needed again here?

Reply:
Yeah that's correct. Removed this one